docs/notebooks/Calculating ground states on large scale systems with the quantum krylov subspace method.ipynb

```diff
@@ -0,0 +1,1037 @@
+      "text/plain": [
+       "<Figure size 296.41x476.583 with 1 Axes>"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "## Create the time-evo op circuit\n",
```

Comment:
You are using the Suzuki-Trotter decomposition here but only introduce it later.